cea/default.config

```diff
@@ -700,6 +700,10 @@ recoverycheckpoint = 0
 recoverycheckpoint.type = IntegerParameter
 recoverycheckpoint.help = in case the optimization stops, it can be resumed from this checkpoint.
 
+random-seed = 64
```

Comment:
I believe the default should be 0 which should be interpreted as not setting the random seed (or setting it to a number derived from "now" - and we print it out when we run a simulation so that if it fails, that can be used in the issue)